.gitignore

```diff
@@ -164,3 +164,4 @@ tmp/
 .prod.env
 .terraform*
 terraform.tfstate
+state.tf
```

Comment:
Should not be in gitignore

Reply:
This was the result of a discussion with Stijn (and some others) concerning how to allow a state.tf file you use locally for testing purposes, as well as an example state.tf file that should be available in the repo.

The conclusion of the discussion was to:
- Create a state.tf.example file, which is just a rename of the previous state.tf file
- Add state.tf to ignore (such that your own local test file is never - accidently - commited to the repo)